lib/modules/datasource/custom/readme.md

```diff
@@ -83,10 +83,14 @@ All available options:
 
 ### Debugging
 
-Renovate writes tracing log entries before transformation starts and after if Renovate finds an unexpected data format.
-To surface this data you can use `logLevelRemap` to surface this data on the App or run Renovate in dryRun mode and log level set to `trace` locally.
+Renovate writes tracing logs entries before transformation.
+If Renovate finds an unexpected dataformat, it also writes a tracing log _after_ transformation.
 
-```json
+#### Getting trace level logs on hosted app
+
+If you use the Mend Renovate app, use the [`logLevelRemap` config option](https://docs.renovatebot.com/configuration-options/#loglevelremap) to get the trace log.
```

Comment:
```suggestion
If you use the Mend Renovate app, use the [`logLevelRemap` config option](../../../self-hosted-configuration.md#loglevelremap) to get the trace log.
```

Reply:
The published docs have `logLevelRemap` as a _repository_ config option. So `self-hosted-configuration.md` is the wrong file, I think.

I'm skipping this suggestion. But we should still put the right link format in! :wink:

Reply:
You are right. Copy and paste error of mine.